WcaOnRails/config/locales/en.yml

```diff
@@ -840,6 +840,15 @@ en:
       competition_announcement_email:
         header: "The WCAT announced %{competition}"
         announced: "The WCAT approved your competition and officially announced it to the public."
+      id_claim_possibility_email:
+        header: "Please link your WCA ID with your account"
+        intro: "Dear Cuber"
+        information_html: >-
+          You registered for %{competition_link}.
+          We just uploaded results for this competition, and created WCA IDs for all first time competitors.
+          If you actually took part in this competition and it was your first one,
+          then you should claim your WCA ID (link it with your WCA account).
+          In order to do this, please visit <a href=%{claim_url}>this page</a>.
```

Comment:
@viroulep this is the only URL left, couldn't find "this page" in `common` so I'm leaving it as is. Is that okay? Or is it better to translate this elsewhere?

Reply:
Yes it would be nice to extract that link too!
We could also adjust the last sentence to something like `You can do this here` :p